src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/PremisesService.kt

```diff
@@ -44,6 +55,23 @@ class PremisesService(
 
   fun getAllPremises(): List<PremisesEntity> = premisesRepository.findAll()
 
+  fun getAllPremisesSummary(serviceName: ServiceName): List<PremisesSummary> {
```

Comment:
So far the api spec and controller set the expectation that this can be used for both services. However this is the point where we pause and focus just on returning temporary-accommodation premises. This is mostly so I can check what I'm doing will work before I keep going and invest in the AP side of things.